### DISCUSSION TITLE
Username and password not accepted on web interface

### DISCUSSION THREAD

**Discussion by dorinf110** [voiceip/oreka#133] [Q&A] [answered]
Hello,
I installed the software on Ubuntu 20.04 LTS (Focal Fossa) using files from ["releases"](https://github.com/voiceip/oreka/releases). The problem is the default username and password are not accepted (admin/admin). I have to mention I installed mariadb instead of MySql5 because mysql5 is no longer available for this release of Ubuntu.  Checking the oreka database I found no table created so maybe this is the cause of malfunctioning. 
Is there a solution to fix the installation and run the software? 

Thank you.

**Comment by kingster** [comment]
I think it is related to the connection to db. Do you get some startup error log in your tomcat logs? Have a look at #56 some versions of  tomcat requires specific escaping characters. 

PS: Sorry for my late reply. somehow missed this thread.

**Reply by dorinf110** [reply]
Thank you for your answer!
 I have a critical error in "catalina.out" log: 
"[2023-07-27 09:27:38] [crit] org.xml.sax.SAXParseException; lineNumber: 43; columnNumber: 38; The value of attribute "password" associated with an element type "wquser" must not contain the '<' character." 
After this error there are a lot of other critical errors and warnings. Do you have any idea how to fix this? Thank you!

**Reply by kingster** [reply]
A quick solve can be creating another MySQL user with no special characters. Can you try this?

**Reply by dorinf110** [reply, marked as answer]
Thank you!
 I solved the problem with MySQL ( I previously made some errors during database configuration). 
Now I can login on orkweb but  I cannot record calls and I have nothing in /var/log/orkweb/. I think the problem is the missing "orktrack.config.xml" file that I was unable to find in github repository. Please, can I get a good version of orktrack.config.xml file?
Thank you!

**Reply by kingster** [reply]
Recording of calls happens though orkaudio. Have you configured the [orkaudio](https://github.com/voiceip/oreka/wiki/Installation#orkaudio)?

**Reply by dorinf110** [reply]
Thank you! Yes, I did it. After searching the internet, I changed localhost to 127.0.0.1 in orkaudio config file and the audio streams are captured and stored in the specified folder. 
Now, the problem is I cannot play those recordings. After pressing the recording play button, the play symbol on the left pane (under the inputs for search) became active for a fraction of a second and after that it deactivates.  Is there something I can check or do in order to make the play working? Thank you very much for your help!

**Reply by dorinf110** [reply]
Sorry for the previous message! All is working fine. The trick was to change the format for the recordings to pcmwav. Than you again for your help!
